spec/lib/web_of_science/services/hash_mappers/names_mapper_spec.rb

```diff
@@ -1,21 +1,15 @@
-describe WebOfScience::MapNames do
+describe WebOfScience::Services::HashMappers::NamesMapper do
```

Comment:
What does this module complexity buy us?  The extra module depth doesn't seem warranted for a relatively small set of files in the existing module.  If this were a much larger module or app, maybe it would be necessary.  As it is, it seems like semantic wrapping (not sure if it has any functional significance).

Reply:
The idea is that we would move every service type class in the Services module. So you could have a Services::Connection, Services::Harvester, and so on

Reply:
Yes, but what does that give us?